tests/kafkatest/benchmarks/core/benchmark_test.py

```diff
@@ -152,8 +156,11 @@ def test_long_term_producer_throughput(self, compression_type="none", security_p
         self.logger.info("\n".join(summary))
         return data
 
+    @cluster(num_nodes=5)
     @parametrize(security_protocol='SSL', interbroker_security_protocol='PLAINTEXT')
-    @matrix(security_protocol=['PLAINTEXT', 'SSL', 'SASL_PLAINTEXT', 'SASL_SSL'], compression_type=["none", "snappy"])
+    @matrix(security_protocol=['PLAINTEXT', 'SSL'], compression_type=["none", "snappy"])
+    @cluster(num_nodes=6)
```

Comment:
Yeah, it's generally only 1 node difference, so there is a chance that it doesn't affect the overall scheduling much if at all.